applications/FluidDynamicsApplication/python_scripts/cfl_output_process.py

```diff
@@ -61,6 +69,11 @@ def __init__(self, model, params):
         # getting output limit for summarization
         self.cfl_output_limit = params["cfl_output_limit"].GetDouble()
 
+        # TODO: Is it ok to do this check? If not, distribution calculation is going to be messy with if conditions for
+        #       case with cfl_output_limit <= 1.0
+        if (self.cfl_output_limit <= 1.0):
+            raise Exception("Please provide cfl_output_limit greater than 1.0")
```

Comment:
I don't understand this... Can you better explain?

Reply:
OK... I think that now I see. Is this the top limit for the statistical distribution?

Reply:
Hi @rubenzorrilla 

We had this discussion when creating the CFL output process. At that time we agreed on CFL Output Limit. It is an output limit, as it say. It is just a double to specify above/below which some values are taken into account when calculating statistical quantitites.